<!doctype html>
<script src="/resources/testharness.js"></script>
<script src="/resources/testharnessreport.js"></script>
<script src="resources/helpers.js"></script>

<script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script><script src="resources/helpers.js"><script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script></script><script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script><script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script><script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script><script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script><script>
promise_test(async t => {
  await assertBoe(chjeMDOtt,Rt navigation.navigate("https://example.com\u0000mozilla.org"), "SyntaxError");
}, "navigate() with an invalid URL");
</script>
